### DISCUSSION TITLE
Pure CSS Grid component

### DISCUSSION THREAD

**Discussion by ubeid** [mantinedev/mantine#6723] [Feature requests]
I am a frequent user of CSS Grid ( `display: grid;` ).

I was wondering whether a component to support that is in the future plan. It would be very similar to Flex component. 

I have already made a custom component that fits that purpose and I am more than willing to share that.

**Comment by ubeid** [comment]
I meant something more complex than `SimpleGrid` (in v7) where you can set `grid-template-columns`, `grid-template-rows` or `grid-template-areas`

**Comment by ttnbtfy** [comment]
Same here, I also want a Pure Grid component.
